tests/results/scale/1.4.0/1.4.0-plus.md

```diff
@@ -0,0 +1,204 @@
+# Results
+
+## Test environment
+
+NGINX Plus: true
+
+NGINX Gateway Fabric:
+
+- Commit: 16a95222a968aef46277a77070f79bea9b87da12
+- Date: 2024-08-16T15:29:44Z
+- Dirty: false
+
+GKE Cluster:
+
+- Node count: 12
+- k8s version: v1.29.7-gke.1008000
+- vCPUs per node: 16
+- RAM per node: 65855012Ki
+- Max pods per node: 110
+- Zone: us-west1-b
+- Instance Type: n2d-standard-16
+
+## Summary:
+
+- Similar results to last time.
```

Comment:
Can we add that this issue still persists: https://github.com/nginxinc/nginx-gateway-fabric/issues/2023